src/Emulator.cpp

```diff
@@ -1,15 +1,13 @@
 #include "PCH.hpp"
 #include "Emulator.hpp"
 
+const Uint32 TimePerFrame = static_cast<Uint32>((1.0 / 60.0) * 1000.0);
```

Comment:
I just made this a constant at the top of the file. It doesn't need to be part of the Emulator, and it's not needed anywhere else. I think this way the compiler will be smarter about it.